TestTools/StreamChatTestTools/Fixtures/JSONs/Thread.json

```diff
@@ -0,0 +1,2400 @@
+{
+  "channel_cid": "messaging:4AB11F2F-4",
```

Comment:
I don't think it's necessary to have such large JSONs for this. Can you reduce them to few entries?

Reply:
Yeah I copy and pasted from a real response, but I can reduce it 👍

Reply:
Done ✅